src/components/BulkEnrollmentPage/stepper/BulkEnrollmentStepper.test.jsx

```diff
@@ -104,12 +104,13 @@ describe('BulkEnrollmentStepper', () => {
     expect(screen.queryByTestId(PREV_BUTTON_TEST_ID)).toBeInTheDocument();
     await act(() => mockEmailResponse);
   });
-  it('clicking previous from Add learners brings you back to Add courses', async () => {
+  it('clicking previous from Add learners brings you back to Add courses and retains selection', async () => {
     renderWithRouter(<StepperWrapper {...defaultProps} />);
     navigateToAddLearners();
     const prevButton = screen.getByTestId(PREV_BUTTON_TEST_ID);
     userEvent.click(prevButton);
     expect(screen.getAllByText(ADD_COURSES_TITLE)).toHaveLength(2);
+    expect(screen.getByText('All 2 selected')).toBeInTheDocument();
```

Comment:
I still cannot for the life of me figure out what the 2 courses are. The debug printout doesn't contain the 2 levels deep where this stuff is generated, but it is definitely looking for 2 courses and if I swap above to selecting one I do need to swap this to `1 selected.` so it is working as intended...I just don't see courses being mocked here, so I'm confuuuuuused. 😩

Reply:
The courses are mocked in the `src/__mocks__` folder! I'm mocking all the Algolia components.